trim_galore/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Docker container for TrimGalore
+
+FROM ubuntu:latest
+
+LABEL software="TrimGalore"
+LABEL software.version="0.6.7"
+LABEL description="Trim Galore is a wrapper around Cutadapt and FastQC to consistently apply adapter and quality trimming to FastQ files, with extra functionality for RRBS data."
+LABEL website="https://github.com/FelixKrueger/TrimGalore"
+LABEL license="GPLv3"
+
+# Deps
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt update && \
+    apt upgrade -y && \
+    apt install -y \
+        build-essential \
```

Comment:
I don't think you need build-essential and python3-dev, even though the documentation suggest you need that (https://cutadapt.readthedocs.io/en/stable/installation.html#dependencies). 
It appears that the is a binary (*.whl - wheel) version of the package for cutadapt (https://pypi.org/project/cutadapt/#files).
Can you give it a try?. It will make the container a bit smaller